synapseclient/core/credentials/credential_provider.py

```diff
@@ -100,7 +102,7 @@ def _get_auth_info(self, syn, user_login_args):
             username = config_username
             password = config_dict.get('password')
             api_key = config_dict.get('apikey')
-            auth_token = config_dict.get('authToken')
+            auth_token = config_dict.get('authtoken')
```

Comment:
Could you also add an entry for `authtoken` to the example config file and remove the one for `apikey`: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/.synapseConfig


Sorry for missing this in the last review!

Reply:
Created a separate PR for the doc https://github.com/Sage-Bionetworks/synapsePythonClient/pull/806